samples/bluetooth/hci_ipc/nrf5340_cpunet_bis-bt_ll_sw_split.conf

```diff
@@ -72,6 +72,7 @@ CONFIG_BT_CTLR_ADV_DATA_BUF_MAX=6
 CONFIG_BT_CTLR_SCAN_AUX_SET=1
 
 CONFIG_BT_CTLR_ADV_RESERVE_MAX=n
+CONFIG_BT_CTLR_ADV_AUX_SLOT_WINDOW=y
```

Comment:
Since this is being enable in so many places, should it be enabled by default?

Reply:
Realized that it has behavioral changes to non-ISO usecases when enabled by default. As ISO features in the Controller are experimental, it is best to use this feature only with ISO builds.

Reply:
We can default it for ISO then?